src/Services/Air/Air.js

```diff
@@ -259,6 +259,33 @@ module.exports = (settings) => {
       return service.flightInfo(parameters);
     },
 
+    async getTicketFromTicketsList(pnr, ticketNumber) {
+      const tickets = await this.getTickets({ pnr });
+      return tickets.find(t => t.ticketNumber === ticketNumber);
+    },
+
+    async getTicketDuplicateTicketFoundCase({
+      pnr,
+      ticketNumber,
+      allowNoProviderLocatorCodeRetrieval
+    }) {
+      const { splitBookings } = await this.getBooking({ pnr });
+      if (!splitBookings) {
```

Comment:
As we discussed, this error should be thrown only if case of split booking. Instead of applying another workaround handler I'd like the code to throw an error when DuplicateTicket error is there, but no split bookings found.

I think it's better to return null here, so original error will be thrown